configure.ac

```diff
@@ -20,8 +20,9 @@
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
 
+
 AC_PREREQ([2.69])
-AC_INIT([cc-oci-runtime], [2.1.7])
+AC_INIT([cc-oci-runtime], m4_esyscmd_s([cat versions.txt  | grep cc_oci_runtime_version | cut -d'=' -f2]))
```

Comment:
I think in this specific part of the configure.ac there is no `$srcdir` variable defined, it's the very beginning of `configure.ac`